test/parallel/test-net-connect-options-fd.js

```diff
@@ -33,8 +33,9 @@ const forAllClients = (cb) => common.mustCall(cb, CLIENT_VARIANTS);
 
 // Test Pipe fd is wrapped correctly
 {
-  const prefix = path.relative(`${common.PIPE}-net-connect-options-fd`,
-                               `${__dirname}-net-connect-options-fd`);
+  // Using relative path on osx if the path is greaterthan 108 chars
+  // an AssertionError: -48 is thrown
```

Comment:
-48 is the libuv error for UV_EADDRINUSE. I don't think this is OS X  specific. It will happen on anything (or at least anything POSIX?) with a long enough path. I think you can just remove the comment (no one's going to wonder "why aren't they using an absolute path here??!!") or use a comment like this:

> // Use relative path to avoid hitting 128-char length limit for socket paths in libuv.

/cc @addaleax to make sure I got that right (that it's 128 chars, that it's in libuv, etc.).